docs/packaging/submitting-a-package.md

```diff
@@ -84,7 +82,7 @@ You will first need to fork the repository either from the GitHub web interface.
 
 Next change your push url to the one that matches your fork. For example
 
-If you forked github.com/solus-packages/nano. It would be forked to github.com/mygithubaccount/nano. You can then set the push url to
+If you forked github.com/solus-packages/nano. It would be forked to `github.com/mygithubaccount/nano`. You can then set the push url to
```

Comment:
The first sentence is a fragment. Recommend:
If you forked github.com/solus-packages/nano, it would be ...